### PR TITLE
COMMONCOMP-199: Remove travis-ci setup

### PR DESCRIPTION
**Pull request by ketul-snps**
MR to remove travis-ci for CI/CD. We will be relying on Jenkins for this purpose.

### CODE REVIEW

**Comment by ketul-snps** [comment]
Also, there are setting changes made to this repository to require at least 1 approval before merging anything to the master branch. The travis-ci checks have been removed from the repository settings.

**Review by vasharapov — APPROVED** [review]

**Review by vigneshd-snps — APPROVED** [review]

**Review by bd-skatzman — APPROVED** [review]

**Review by gopannabd — APPROVED** [review]

**Pull request merged**
